src/Module/Api/ApiResponse.php

```diff
@@ -108,14 +109,16 @@ public function createXML(array $data, string $root_element): string
 	 * @param string $root_element Name of the root element
 	 * @param string $type         Return type (atom, rss, xml, json)
 	 * @param array  $data         JSON style array
+	 * @param int    $cid          ID of the contact for RSS
 	 *
 	 * @return array|string (string|array) XML data or JSON data
 	 */
-	public function formatData(string $root_element, string $type, array $data)
+	public function formatData(string $root_element, string $type, array $data, int $cid = 0)
 	{
 		switch ($type) {
-			case 'atom':
 			case 'rss':
+				$data = BaseApi::addRSSValues($data, $cid);
```

Comment:
This would lead to a possible, cyclic, dependency (`BaseApi` is the mother class of any `Api`Module, which calls `ApiResponse`, which is calling `BaseApi` again)

Please move `BaseApi::addRSSValues()` to this class as a private, dynamic class.
Add `twitterUser` and `BaseUrl` as protected variables inside the constructor to avoid a `DI::` call here.